datastore.go

```diff
@@ -52,7 +59,9 @@ var DefaultOptions Options
 
 func init() {
 	DefaultOptions = Options{
-		gcDiscardRatio: 0.1,
+		GcDiscardRatio: 0.5,
```

Comment:
https://github.com/dgraph-io/badger/blob/master/db.go#L1069

Reply:
@hsanjuan I see. Badger recommends a value of 0.5 in the docs with the premise that a lower value will cause more GC's to run. But, I wasn't aware that we don't usually fill up 50 % of a value log file.  I'll change it to a lower value.